app/code/Ludmila/LSNewClasses/Model/ShowFiles.phtml

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Ludmila\LSNewClasses\Model;
+
+class ShowFiles
+{
+    public function show()
+    {
+        $path = realpath('/misc/apps/magento-226/app/code/Ludmila/LSNewClasses');
```

Comment:
This path could be different on other workspace. Please inject \Magento\Framework\Filesystem\DirectoryList and use getRoot() method from it